### PR TITLE
Support to deploy CBL release builds

### PR DESCRIPTION
**Pull request by hrajput89**
#### Fixes #.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

### CODE REVIEW

File: keywords/TestServeriOS.py [hunk 1]
```diff
@@ -73,12 +91,18 @@ def download(self, version_build=None):
         # Package not downloaded, proceed to download from latest builds
         downloaded_package_zip_name = "{}/{}".format(BINARY_DIR, self.package_name)
         if self.platform == "ios":
-            url = "{}/couchbase-lite-ios/{}/{}/{}".format(LATEST_BUILDS, version, build, self.package_name)
+            if self.build is None:
+                if self.version < "2.0.2":
```
Comment (sridevi-15) [inline keywords/TestServeriOS.py]: both if and else stmts are same, why is this if condition required?

File: keywords/TestServeriOS.py [hunk 2]
```diff
@@ -73,12 +91,18 @@ def download(self, version_build=None):
...
+                    url = "{}/couchbase-lite-ios/{}/{}".format(RELEASED_BUILDS, self.version, self.package_name)
+            else:
+                url = "{}/couchbase-lite-ios/{}/{}/{}".format(LATEST_BUILDS, self.version, self.build, self.package_name)
         else:
-            url = "{}/couchbase-lite-net/{}/{}/{}".format(LATEST_BUILDS, version, build, self.package_name)
+            url = "{}/couchbase-lite-net/{}/{}/{}".format(LATEST_BUILDS, self.version, self.build, self.package_name)
 
         log_info("Downloading {} -> {}/{}".format(url, BINARY_DIR, self.package_name))
-        resp = requests.get(url)
+        resp = requests.get(url, verify=False)
```
Comment (sridevi-15) [inline keywords/TestServeriOS.py]: I hope this is for to go through https url, right?

File: keywords/TestServeriOS.py [hunk 3]
```diff
@@ -122,7 +146,14 @@ def install(self):
             if self.debug_mode:
                 self.app_name = "{}-{}-debug.app".format(self.app, self.version_build)
```
Comment (sridevi-15) [inline keywords/TestServeriOS.py]: we should add for debug_mode too

File: keywords/TestServerAndroid.py [hunk 4]
```diff
@@ -49,19 +49,24 @@ def download(self, version_build=None):
             self.version_build = version_build
         version, build = version_and_build(self.version_build)
 
+        if version < "2.1.2":
```
[the remark this answers is not in the document]
Comment (hrajput89) [reply keywords/TestServerAndroid.py]: app is not build for below version, check raise statement

File: keywords/TestServerNetMsft.py [hunk 5]
```diff
@@ -48,14 +48,22 @@ def __init__(self, version_build, host, port, community_enabled=None, platform="
         self.version_build = version_build
         self.version, self.build = version_and_build(self.version_build)
 
+        if version_build <= "2.1.0":
```
[the remark this answers is not in the document]
Comment (hrajput89) [reply keywords/TestServerNetMsft.py]: app is not build for below version, check raise statement

File: keywords/TestServeriOS.py [hunk 6]
```diff
@@ -29,7 +30,15 @@ def __init__(self, version_build, host, port, community_enabled=None, debug_mode
         self.app_name = ""
         self.platform = platform
         self.bundle_id = ""
-
+        self.released_version = {
```
[the remark this answers is not in the document]
Comment (hrajput89) [reply keywords/TestServeriOS.py]: naming convention is not consistent across all platform. It works for all platform.

File: keywords/TestServeriOS.py (hunk 1 above)
Comment (hrajput89) [reply keywords/TestServeriOS.py]: not they are not - "couchbase-lite/ios" v/s "couchbase-lite-ios"

File: keywords/TestServeriOS.py (hunk 2 above)
Comment (hrajput89) [reply keywords/TestServeriOS.py]: for certificate validation

File: keywords/TestServeriOS.py (hunk 3 above)
Comment (hrajput89) [reply keywords/TestServeriOS.py]: add what?

File: keywords/TestServeriOS.py [hunk 7]
```diff
@@ -29,7 +30,15 @@ def __init__(self, version_build, host, port, community_enabled=None, debug_mode
...
+            "2.0.0": 806,
+            "2.1.0": 263,
+            "2.1.1": 10,
+            "2.1.2": 11,
+        }
+
+        if self.version_build == "2.1.0":
```
Comment (sridevi-15) [inline keywords/TestServeriOS.py]: do you mean < "2.1.0" or == is intentional. If == is intentional, then ios build available  which are below "2.1.0" builds?

File: keywords/TestServeriOS.py (hunk 3 above)
Comment (sridevi-15) [reply keywords/TestServeriOS.py]: the logic if build is not provided , get the build from latest builds. You have added that for release mode, but not for debug mode

File: keywords/TestServeriOS.py (hunk 7 above)
Comment (hrajput89) [reply keywords/TestServeriOS.py]: app is not available for this version, hence ==

File: keywords/TestServeriOS.py (hunk 3 above)
Comment (hrajput89) [reply keywords/TestServeriOS.py]: Added logic for debug mode.

**Pull request merged**